portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,7 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
Maybe import just the classes that you need? 

https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Fixed this!